tests/unit/test_charm.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+"""Test default charm events such as upgrade charm, install, etc."""
+
+import unittest
+from unittest.mock import PropertyMock, patch
+
+import ops.testing
+from charm import SlurmdbdCharm
+from ops.model import ActiveStatus, BlockedStatus
+from ops.testing import Harness
+
+ops.testing.SIMULATE_CAN_CONNECT = True
+
+
+class TestCharm(unittest.TestCase):
+    def setUp(self) -> None:
+        self.harness = Harness(SlurmdbdCharm)
+        self.addCleanup(self.harness.cleanup)
+        self.harness.begin()
+
+    @unittest.expectedFailure
+    def test_install_success(self) -> None:
+        """Test that slurmdbd can successfully be installed.
+
+        Notes:
+            This method is expected to fail due to the 'version' file missing.
```

Comment:
I do think the test name and the docstring itself don't reflect in good what the test expects. Possibly `test_install_failure` is more realistic. I would suggest for the docstring something like `Test whether slurmdbd installation is failed due to missing version file`.

The test name can even be something like `test_install_failure__missing_version_file` then the docstring is useless indeed. Anyway, just a suggestion for improvement :)

Reply:
As I had mentioned in my [previous comment](https://github.com/omnivector-solutions/slurmdbd-operator/pull/1/files/f97a529f18f374933632bff1a5b5d5280c0a89f3#r1068220256), this is just temporary. I will make it more clear in another pull request that I am testing this block of code:

https://github.com/omnivector-solutions/slurmdbd-operator/blob/78eb66ee34dfadd86a9e93f76a60e174fb17853b/src/charm.py#L85-L94

Thank you for pointing this out!